From 1b3d450833e4a9ff62c50582782c3a2fe7553cfa Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Sat, 5 Dec 2020 20:41:40 +0530
Subject: [PATCH 12/40] Revert "ANDROID: CC_FLAGS_CFI add
 -fno-sanitize-blacklist"

This reverts commit e221e894adeb6227a7e6eb0974c44cc2c68c4ab8.
---
 Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 128d3bbd6789..1aff2d4b7ff9 100644
--- a/Makefile
+++ b/Makefile
@@ -907,8 +907,7 @@ endif
 
 ifdef CONFIG_CFI_CLANG
 CC_FLAGS_CFI	:= -fsanitize=cfi \
-		   -fno-sanitize-cfi-canonical-jump-tables \
-		   -fno-sanitize-blacklist
+		   -fno-sanitize-cfi-canonical-jump-tables
 
 ifdef CONFIG_MODULES
 CC_FLAGS_CFI	+= -fsanitize-cfi-cross-dso
-- 
2.17.1

